README.md

```diff
@@ -33,15 +40,9 @@ See also dockerhub tags page: https://hub.docker.com/r/fluent/fluentd-kubernetes
 - `Cloudwatch`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-cloudwatch-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-cloudwatch-1`
-- `Stackdriver`
-  - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-stackdriver-1.0`
-  - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-stackdriver-1`
 - `S3`
```

Comment:
s3 is not multiarched. Only for amd64:
https://hub.docker.com/layers/fluent/fluentd-kubernetes-daemonset/v1.13.1-debian-s3-amd64-1.0/images/sha256-b5a8dd30151795a25e477a34152e1d177ba8492098f8783c0f4ab12e18dd56da?context=repo

Reply:
fixed